readme.md

```diff
@@ -44,68 +46,38 @@ console.log(stdout);
 
 ### Scripts interface
 
-For more information, please see [this page](docs/scripts.md).
+For more information about Execa scripts, please see [this page](docs/scripts.md).
 
 #### Basic
 
 ```js
 import {$} from 'execa';
 
-const {stdout} = await $`echo unicorns`;
-// const {stdout} = await $`echo ${'unicorns'}`;
-// const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
+const unicorns = await $`echo unicorns`;
+const {stdout} = await $`echo ${unicorns}${'!'}}`;
+console.log(stdout);
+//=> 'unicorns!'
 
+const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
 console.log(stdout);
-//=> 'unicorns'
+//=> 'unicorns rainbows'
```

Comment:
This example errors on reassignment of block scoped var `stdout`

Reply:
Right! Fixed.